governance/votes/Executive vote - January 24, 2022.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Template - [Executive Vote] Temporarily prevent FLAP auctions and MKR Burn - January 24, 2022
+summary: Increase the System Surplus Buffer to 250 million DAI to prevent FLAP auctions until the auction/burn mechanism can be improved.
+date: 2022-01-24T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Temporarily prevent FLAP auctions and MKR Burn - January 24, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an out-of-schedule executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+This is an out-of-schedule executive proposal. It has been deployed and added to the voting portal in response to the large wave of FLAP auctions triggered by liquidations at the end of last week and the following [urgent signal request](https://forum.makerdao.com/t/urgent-signal-request-stop-mkr-burning/12806). 
+
+The express purpose of this proposal is to prevent FLAP auctions temporarily until the auction/burn mechanism can be improved.
+
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+* The existing [System Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) lerp function will be disabled.
+* The [System Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) parameter will be set to 250 million DAI.
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Temporarily prevent FLAP auctions and MKR Burn
+
+As per this [urgent signal request](https://forum.makerdao.com/t/urgent-signal-request-stop-mkr-burning/12806), the following changes will be made if this executive proposal passes.
+* The existing [System Surplus Buffer](https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer) lerp function will be disabled.
```

Comment:
Do we need to elaborate on what lerp is/link the previous votes for it?

Reply:
Uhm, maybe, we don't have good docs for lerp, and there are already so many links...